pkgs/development/python-modules/monai/default.nix

```
@@ -0,0 +1,63 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, ninja
+, ignite
+, numpy
+, pybind11
+, pytorch
+, which
+}:
+
+buildPythonPackage rec {
+  pname = "monai";
+  version = "0.9.0";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "Project-MONAI";
+    repo = "MONAI";
+    rev = "${version}";
+    sha256 = "sha256-HxW9WYxt2a7fS9/1E9DtiH+SCTTJoxYBfgZqskYdcvI=";
+  };
+
+  patchPhase = ''
```

```suggestion
  postPatch = ''
```

No need to override the phase.

---

Fixed.